integration/integrate-with-email.adoc

```diff
@@ -7,17 +7,33 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 [role="_abstract"]
-Configure {product-title} ({product-title-short}) to send alerts about policy violations to a standard email provider.
+With {product-title} ({product-title-short}), you can configure your existing email provider and send notifications about policy violations. Or, if you are using {rh-rhacscs-first}, you can send email notifications using the built-in email notifier.
 
-You can use email as a notification method by forwarding alerts from {product-title-short} to a standard email provider. To forward alerts from the {product-title-short} platform to an email address, you can use the `Default Recipient` field to send email to a standard and centralized team, or use deployment annotations to specify an audience for notifications.
+To forward alerts from the {product-title-short} {product-title-managed-short} platform to an email address, you can use the `Default recipient` field to send emails to a standard and centralized team or use deployment annotations to specify an audience for notifications.
 
-With annotation keys, you can define an audience to notify about policy violations that are associated with a deployment or namespace. If the deployment has an annotation, the annotation's value overrides the default value. If the namespace has an annotation, the namespace's value overrides the default value.
+With annotation keys, you can define an audience to notify about policy violations associated with a deployment or namespace.
```

Comment:
We are not allowed to add hyperlinks in modules, so if we want to add a link back to here, we would have to add it under the Additional resources section after the modules, which defeats the purpose, IMO. However, since we are talking about annotations in the module, we can add this text to both modules. For example where we talk about annotations in `Configuring the email plugin`, we can add this bit.